index.js

```diff
@@ -264,7 +269,7 @@ window.mobileCheck = function() {
                     }
                 })
             }
-            centreData = Object.assign({}, {name:centre["name"]}, daysData);
+            centreData = Object.assign({}, {name:centre["name"], name_district:centre["name"]+" ("+centre["district"]+")"}, daysData);
```

Comment:
Rather than constructing string to render here, just add required data, and use "render:" of data tables. Search this source for "render:" for sample usage.